Modules/TOF/CMakeLists.txt

```diff
@@ -2,16 +2,29 @@
 
 add_library(QcTOF)
 
-target_sources(QcTOF PRIVATE src/TOFTask.cxx src/TOFCheckRawsMulti.cxx
-        src/TOFCheckRawsTime.cxx src/TOFCheckRawsToT.cxx)
+target_sources(QcTOF 
+              PRIVATE src/TOFTask.cxx
+                      src/TOFTaskCompressed.cxx
+                      src/TOFCheckDiagnostic.cxx
+                      src/TOFCheckRawsMulti.cxx
+                      src/TOFCheckRawsTime.cxx 
+                      src/TOFCheckRawsToT.cxx)
 
 target_include_directories(
   QcTOF
   PUBLIC $<INSTALL_INTERFACE:include>
          $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-  PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
-
-target_link_libraries(QcTOF PUBLIC QualityControl)
+  PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src
+         ${O2_ROOT}/include/TOFWorkflow
```

Comment:
This should not be needed if I am not mistaken. The imported targets defined later in `target_link_libraries` are enough.

Reply:
Dear Barthelemy, indeed, as you suggested I modified the CMakeLists file